OpenQA/Isotovideo/Utils.pm

```diff
@@ -101,7 +101,7 @@ sub _open_cache_index ($root_cache_dir, $index_file) {
 
 sub _determine_size ($dir, $handle_output) {
     $handle_output->($?, my $du = qx{du -s "$dir"});
-    die "Unable to determine size of Git directory under '$dir': du returned '$du'\n" unless $du =~ /(\d+).*/;
+    die "Unable to determine size of Git directory under $dir : du returned '$du'\n" unless $du =~ /(\d+).*/;
```

Comment:
The directory shouldn't be problematic so I guess we shouldn't change this line. The same counts for the other occurrences of `$dir`.

Reply:
As I see there are cases where `$dir` is actual a URL. I dont know if the name can change tho

Reply:
Do you have an example where `$dir` is a URL? The variable name sounds wrong in that case.

Reply:
I'm pretty sure this function expects a normal path. If `$dir` was a URL this function would not work correctly and should not be called in the first place. (I don't think this is the case, though. Probably you can really just omit this change.)